src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -197,17 +176,23 @@ const NavHeader = (props) => {
                   {isFollow === undefined && isLoading ? (
                     <Shimmer width={normalize(100)} height={normalize(36)} />
                   ) : isFollow ? (
-                    <ButtonFollowing handleSetUnFollow={handleFollowTopic} />
+                    <ButtonFollowing
+                      handleSetUnFollow={onFollowButtonPress}
+                      followType={followType}
+                    />
                   ) : (
-                    <ButtonFollow handleSetFollow={handleFollowTopic} />
+                    <ButtonFollow handleSetFollow={onFollowButtonPress} />
                   )}
                 </Animated.View>
               </View>
             </View>
             <Animated.View
               style={[
                 styles.domain(isHeaderHide),
-                {bottom: normalize(isFollow ? 13 : 24), opacity: opacityHeaderAnimation}
+                {
+                  bottom: normalize(isFollow && followType === 'signed' ? 13 : 24),
+                  opacity: opacityHeaderAnimation
+                }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `onFollowButtonPress` and `followType` in `ButtonFollowing` and `ButtonFollow` components is consistent with the changes made earlier. This maintains consistency across your codebase.



<!-- This is an auto-generated comment by OSS CodeRabbit -->